LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Thomas Vindard
```

Comment:
Don't know how Licenses work but need all Copyright collaborator names ? idk

Reply:
Don't know why this is here as it is already in the repo! According to the MIT Licence, the name should be the person or the organisation hosting the repository so as it is my account I only put my name... I have no right on the code with this licence don't worry ;)

Will check with Laurent